README.md

```diff
@@ -47,16 +47,14 @@ linter after my tests I would do so like this.
 [dogma]: https://github.com/lpil/dogma
 
 ```elixir
-# config/config.exs
+# config/dev.exs
```

Comment:
This won't work in most cases, only Phoenix projects have `config/dev.exs` included by default.